### PR TITLE
Board Report 2023/02

### PR DESCRIPTION
**Pull request by spier**
Quick link: [markdown](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/board-report-2023-02/meta/boardreports/2023-02.md)

### CODE REVIEW

File: meta/boardreports/2023-02.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    * [Extensions to Manage Contributions at Scale](https://patterns.innersourcecommons.org/p/extensions-for-sustainable-growth) - great work @Ssukriti
+* New **Initial** patterns:
+    * [Incubator Pipeline](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/patterns/1-initial/incubator-pipeline.md) - thanks @robtuley for moving this long-standing PR along and getting the content into our repo for greater visibility of the ISC community
+    * [InnerSource Customer Interview Questions](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/patterns/1-initial/innersource-customer-interview-questions.md) - contributed by @kschueths as part of the last Summit. Thank you!
+* Adding Known Instances to various patterns e.g. WellSky, Dazn, and Mercado Libre. Special thanks to @privero-meli from our community in Brazil for adding Mercado Libre.
+* We are trying out GitHub Releases ([example](https://github.com/InnerSourceCommons/InnerSourcePatterns/releases/tag/v1.4)) to highlight all contributors to our book. We hope that they will appreciate that acknowledgement, and continue their contributions.
+* Various minor updates to existing patterns.
+
```
Comment (yuhattor) [inline meta/boardreports/2023-02.md:47]: I don't know if this is the right place to write this, but I'll do it for the record.
```suggestion
* Recorded 20 instructional YouTube videos on InnerSource Patterns in Japanese. All will be rolled out in Q1 and Q2 of CY2023. It is hoped that the InnerSource Patterns page and the Japanese page will benefit from improved SEO.

```

Comment (spier) [reply meta/boardreports/2023-02.md:47]: This is great!
Which link can I use to point the "20 instructional YouTube videos on InnerSource Patterns in Japanese"?

Comment (spier) [reply meta/boardreports/2023-02.md:47]: I think I found the playlist, will add it to the report.
https://www.youtube.com/watch?v=rbfXGedu81g&list=PLzsDnO3v-utv349AtbxElYAqrXN6XZVqR

However note that YouTube shows me "12 unavailable videos are hidden". 
Probably those 12 of the 20 videos are not made public yet?

Comment (spier) [reply meta/boardreports/2023-02.md:47]: My bad, I did not read your statement correctly
> All will be rolled out in Q1 and Q2 of CY2023

Still odd that YouTube tells me that there are more videos in the playlist than what I can actually watch :)

**Pull request merged**